contracts/Market.sol

```diff
@@ -891,7 +900,7 @@ contract Market is
     uint256 restrictedSupplierFee;
     uint256 unrestrictedSupplierFee;
     for (uint256 i = 0; i < countOfRemovalsAllocated; ++i) {
-      unrestrictedSupplierFee = removalAmounts[i];
+      unrestrictedSupplierFee = (removalAmounts[i] * _priceMultiple) / 100;
```

Comment:
Use muldiv

Reply:
Also wonder if there is value in making this an external view function

Reply:
you mean a function for `unrestrictedSupplierFee` ?  I'm not sure how much sense that would make out of context or where else we'd want to use it